msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Designing your meeting extension
```

Comment:
Not an edit in this PR but I notice that the article uses Gerund in the title. Is this OK as per the style guide?

Reply:
Updated the Title